### PR TITLE
#147: Version bump

### PR DESCRIPTION
**Pull request by GitRon**

### CODE REVIEW

**Comment by smithdc1** [comment]
Looks good. 👍

What do you think about bumping this classifier to 5 - Production/stable?

https://github.com/django-crispy-forms/crispy-tailwind/blob/main/pyproject.toml#L13

https://pypi.org/classifiers/

**Comment by GitRon** [comment]
@smithdc1 Great idea! I bumped the version to 1.0 to reflect the prod classifier.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # CHANGELOG for crispy-tailwind
 
 ## Next Release (tbc)
+
+## 0.6 (2024-01-09)
```
Comment (smithdc1) [inline CHANGELOG.md:5]: Typo here given change to bump to 1.0. 

```suggestion
## 1.0 (2024-01-09)
```

Comment (GitRon) [reply CHANGELOG.md:5]: Oh damn! Fixed it.

**Pull request merged**